### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by fosterlynn**
Updating the doc to help with NextEdge work.  Review from @ahdinosaur and @pmackay would be greatly appreciated, with @elf-pavlik out of pocket.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,56 +7,102 @@ Vocabulary to describe networks of people and groups.
...
 #### Properties
 
 * @id ([JSON-LD](http://www.w3.org/TR/json-ld/) specific way of setting [rdf:subject](http://www.w3.org/TR/rdf-schema/#ch_subject))
 * @type ([JSON-LD](http://www.w3.org/TR/json-ld/) specific alias of [rdf:type](http://www.w3.org/TR/rdf-schema/#ch_type))
-* url 
-* displayName / displayNameMap
-* image
-* (will be others)
+* url (a URL)
+* displayName (skos:prefLabel)
```
Comment (ahdinosaur) [inline README.md:100]: `displayName` should be `label`, also we should add `note` to all the entities as well.

Comment (ahdinosaur) [inline README.md:100]: based on https://github.com/valueflows/valueflows/blob/master/context.jsonld#L11-L14

**Comment by ahdinosaur** [comment]
looks great @fosterlynn. :smile: made a comment about `label` and `note`.

File: README.md (hunk 1 above)
Comment (fosterlynn) [inline README.md:100]: @ahdinosaur thanks, was having a hard time finding final decisions on some of these, makes sense it is in the context.  Get it on note.  

On label, did we decide to use label for people's names as well as group names?  Doesn't seem right to me now.  I get it if everything is just a graph on a page, but seems not so good for a domain vocab.  (Sorry probably am bike shedding again.... or maybe all vocab work is that way? :)  )

Comment (ahdinosaur) [inline README.md:100]: yeah, i feel this. is there something that works for both? (i really like simplicity and consistency.) we want to represent `skos:prefLabel`, `skos:altLabel`, and `skos:hiddenLabel`. so maybe we do `vf:name` (as in [Activity Streams](http://www.w3.org/TR/activitystreams-vocabulary/#dfn-name) and [schema.org](http://schema.org/name)), `vf:alternateName` (as in [schema.org](http://schema.org/alternateName)), and `vf:hiddenName`?

Comment (fosterlynn) [inline README.md:100]: Well I thought vf:displayName wasn't too bad for both people and groups, and it was on elf's example json-ld.  :)  vf:name is fine with me too. I definitely agree it has to be the same for all kinds of agents.  And that it should represent skos:prefLabel.  Or are you thinking it should be the same for all nodes? Still the name stuff might work for that too.  I did change it to label for now.  Maybe we can see what the first user thinks? Or just make an executive decision without elf?

Comment (ahdinosaur) [inline README.md:100]: i mean all nodes. pulled out into a proposal: https://github.com/valueflows/valueflows/issues/101.

**Pull request merged**